src/npctalk.cpp

```diff
@@ -632,7 +632,7 @@ std::string dialogue::dynamic_line( const talk_topic &the_topic ) const
     }
 
     if( topic == "TALK_NONE" || topic == "TALK_DONE" ) {
-            return "";
+            return "Bye.";
```

Comment:
Needs to be translated. Add `_(` and `)` around the string, like it's done in the code below.